pkg/didcomm/messaging/service/http/httpmsgsvc_test.go

```diff
@@ -10,35 +10,353 @@
 package http
 
 import (
+	"encoding/json"
```

Comment:
I assume BDD tests are coming in a later PR (demonstrating HTTP calls).

Reply:
Yes